platformio.ini

```diff
@@ -779,3 +780,15 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+
+# ------------------------------------------------------------------------------
+# NetCube Systems Universal Light Controller
+# ------------------------------------------------------------------------------
+[env:esp32_ulc]
+board = esp32-poe
+build_unflags = ${common.build_unflags}
+build_flags = ${common.build_flags_esp32} -D WLED_RELEASE_NAME=ESP32_ULC -D RLYPIN=-1 -D WLED_USE_ETHERNET -D WLED_ETH_DEFAULT=6 -D LEDPIN=14 -D DEFAULT_LED_COUNT=37 -D APL_MILLIAMPS_DEFAULT=65000 -D BTNPIN=-1 -D WLED_DISABLE_BLYNK -D IRPIN=-1 -D USERMOD_ULC_BATTERYMANAGEMENT
```

Comment:
> -D APL_MILLIAMPS_DEFAULT=65000
why do you need this? It could be dangerous to set 65 Ampere as current limit.

Reply:
This was left in from testing by accident. It is now set to 2.5A as default for the board.